pkg/network/java/testutil/JavaClientSimulator/Readme.md

```diff
@@ -0,0 +1,23 @@
+JavaClientSimulator allows testing different java **https** frameworks.
+You can run the attached docker or directly by executing the jar package.1
+
+This test util was built using JDK 11
+
+## Configuration
+
+you can control
+- java client framework to use. Currently supported: ApacheHttp, OkHttp, URLConnection, HttpClient
+- target url
+- iterations - number of requests to send. By default infinitely
+- timeout - interval between each iteration. By default 1 second
+
+## Standalone:
+
+Build: `mvn clean package`
+Run:  `java -jar ./target/JavaClientSimulator-1.0.jar client=<client> url=<url>`
```

Comment:
If you want to have them line by line then you must have an empty separate line in between
```suggestion
Build: `mvn clean package`

Run:  `java -jar ./target/JavaClientSimulator-1.0.jar client=<client> url=<url>`
```